test/transforms.jl

```diff
@@ -680,6 +680,34 @@
     @test n1 == n2
   end
 
+  @testset "StdNames" begin
+    a = [3, 2, 1, 4, 5, 3]
+    b = [2, 4, 4, 5, 8, 5]
+    t = Table(; a, b)
+    t = t |> Rename(:a => "apple tree", :b => "banana tree")
```

Comment:
You can create a table with the desired column names instead of using Rename here.

Reply:
Also, in this example make sure that the table also contains clean column names such as `a`, `B`. They shouldn't be affected by some of the conventions.

Reply:
@juliohm  I have tried using a Symbol or a string instead of a variable name, but both of them throw syntax error. I am not sure how I can create a table with these column names.